### DISCUSSION TITLE
ModernGadgets not working on win11

### DISCUSSION THREAD

**Discussion by 19yews** [raiguard/ModernGadgets#286] [Q&A] [answered]
Hello y'all !
Since yesterday rainmeter with the great moderngadgets stopped working; reinstalled rainmeter and moderngadgets caused the same result... replayed a one day old image of win11 and, there it was but, only for some minutes, then disappeared again... played around but always with the same result, moderngadgets won't work no more.
raiguard, could you have a look at it, would be great.
thx in advance,
YA

**Comment by raiguard** [comment]
Could you post the output of the Rainmeter log? We need to check if it's Rainmeter itself crashing, or if something is going wrong in MG.

**Reply by 19yews** [reply]
Sorry, I have no logs saved but, I think that rainmeter crashes with MG and other skins as well

**Reply by raiguard** [reply]
Can you try deleting the Rainmeter.ini file, so Rainmeter will load the default skins?

**Comment by SilverAzide** [comment, marked as answer]
I'm getting several reports of my Gadgets suite crashing Rainmeter all of a sudden, when everything was working fine previously.  The common thread is pointing to the Performance Counter database getting corrupted, possibly due to a recent Windows Update.

Try this:  [PerfMon Troubleshooting](https://github.com/raiguard/ModernGadgets/wiki/PerfMon-Troubleshooting).

**Comment by 19yews** [comment]
Hey raiguard and SilverAzide !

After several tries of deleting the Rainmeter.ini file and trying to run MG anew, I had always the same result, Rainmeter crashing.
Where would I find the Rainmeter logs?

Now, with your hint, SilverAzide, to run the PerfMon Troubleshooting, I was that glad that MG is back again.

For me MG is a treasure of a skin, I looked a lot of others but none comes near in clarity and design, simple and rich at the same time.

I am nearly 72 and use my i7-9700 with 32GB RAM just for fun and conversations, emails and similar.
If you are into blogging I would like to see you on our [Diaspora social network](https://www.wikiwand.com/en/Diaspora_(social_network)) - totally decentralized. There are a lot of linux freaks, programmers, musicians, art guys and many more, from all over the world. You could [choose a pod here](https://the-federation.info/).

And, finally a big thank you for the great help!

With kind regards and greetings from Austria,
Alfred
